nservicebus/operations/tuning.md

```diff
@@ -9,12 +9,12 @@ related:
 - nservicebus/licensing/licensing-limitations
 ---
 
-NServiceBus will by default allow the transport to optimize for maximum performance when it comes to message processing. While this is usually the preferred mode of operation there are situations where tuning needs to be applied.
+NServiceBus will by default try to optimize for maximum performance when it comes to message processing. While this is usually the preferred mode of operation there are situations where tuning needs to be applied.
```

Comment:
That sounds like we have some automatic adjustments built-in in NSB, that's not correct, is it?